packages/flutter/lib/src/rendering/shifted_box.dart

```diff
@@ -741,6 +734,78 @@ class RenderUnconstrainedBox extends RenderAligningShiftedBox with DebugOverflow
   }
 }
 
+/// Renders a box, imposing no constraints on its child, allowing the child to
+/// render at its "natural" size.
+///
+/// This allows a child to render at the size it would render if it were alone
+/// on an infinite canvas with no constraints. This box will then attempt to
```

Comment:
Alone on an Infinite Canvas with No Constraints will be the name of my breakout studio album.